files/default/neutron-ha-tool.py

```diff
@@ -737,10 +737,21 @@ def target_agent_list(agent_list, agent_type, exclude_agent_host):
     :param exclude_agent_host: hostname of agent to exclude from the list
```

Comment:
Added, but I disagree with the etherpad and the justification is weak. For one side nearly all projects refers to bugs using the local prefix associated with namespace of the project, or even with a single number. For the other side our commits from other project do not follow this rule.

>> I am not aware of this team policy etherpad

> You are now, since I told you ;-)

No. Not really. When this policy was discussed? Who participate on it? When has been announced and communicated? I am not able to find this in the mailing list.